src/lib.rs

```diff
@@ -26,6 +26,19 @@
 //!
 //! # Use
 //!
+//! ## Logging levels
+//! 
+//! This library makes use of the following macros:
+//! 
+//! - [error](https://github.com/rust-lang-nursery/log/blob/master/src/macros.rs#L73)
+//! - [warn](https://github.com/rust-lang-nursery/log/blob/master/src/macros.rs#L104)
+//! - [info](https://github.com/rust-lang-nursery/log/blob/master/src/macros.rs#L138)
+//! - [debug](https://github.com/rust-lang-nursery/log/blob/master/src/macros.rs#L172)
+//! - [trace](https://github.com/rust-lang-nursery/log/blob/master/src/macros.rs#L209)
+//! 
+//! All of these macros accept formatting strings
+//! similarly to [println!](https://github.com/rust-lang/rust/blob/master/src/libstd/macros.rs#L127)
```

Comment:
Thanks for the update @brson. Sorry not to reply sooner.

I have converted it to prose version and updated the links.